app/src/test/java/com/codesoom/assignment/application/AuthenticationServiceTest.java

```diff
@@ -0,0 +1,151 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import com.codesoom.assignment.dto.UserLoginData;
+import com.codesoom.assignment.errors.InvalidAccessTokenException;
+import com.codesoom.assignment.errors.LoginNotMatchPasswordException;
+import com.codesoom.assignment.errors.UserNotFoundException;
+import com.codesoom.assignment.utils.JwtUtil;
+import com.github.dozermapper.core.DozerBeanMapperBuilder;
+import com.github.dozermapper.core.Mapper;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("AuthenticationService 테스트")
+@DataJpaTest
+class AuthenticationServiceTest {
+
+
+    @Autowired
+    private UserRepository userRepository;
+
+    private Mapper mapper;
+    private JwtUtil jwtUtil;
+
+    private static final String SECRET = "12345678901234567890123456789010";
+    private AuthenticationService authenticationService;
+
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        user = User.builder()
+                .email("test@test.com")
+                .password("password")
+                .name("name").build();
+
+        mapper = DozerBeanMapperBuilder.buildDefault();
+        jwtUtil = new JwtUtil(SECRET);
+        authenticationService = new AuthenticationService(jwtUtil, mapper, userRepository);
+    }
+
+    @AfterEach
+    void tearDown() {
+        userRepository.deleteAll();
+    }
+
+    @Nested
+    @DisplayName("login 메서드")
+    class Describe_login {
+
+        @Nested
+        @DisplayName("올바른 유저 정보가 주어지면")
+        class Context_with_correct_user {
+
+            private User givenUser;
+            private String valid_token;
+
+            @BeforeEach
+            void prepare() {
+                givenUser = userRepository.save(user);
+                valid_token = jwtUtil.encode(givenUser.getId());
+            }
+
+            @Test
+            @DisplayName("token을 반환합니다.")
```

Comment:
이렇게 설명하면 `prepare`와 잘 어울리겠죠.
```suggestion
            @DisplayName("jwt로 인코딩된 token을 반환합니다.")
```